### PR TITLE
Testing a bit

### PR DESCRIPTION
**Pull request by zendern**
Just started to write some unit tests around the rules. 
Also plan on doing an integration test as well and hitting a real db as part of this PR.
And figure once they are there i'll do like extract-pg-schema and wire up github actions as part of this too.

### CODE REVIEW

File: src/rules/nameCasing.test.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import {nameCasing} from "./nameCasing";
+
+describe('nameCasing', () => {
+    it("no tables or views passed no errors", () => {
```
Comment (kristiandupont) [inline src/rules/nameCasing.test.js:4]: It looks like your editor defaults to 4 spaces indenting. Are you running prettier?

Comment (zendern) [reply src/rules/nameCasing.test.js:4]: I use intellij so it's just the defaults. I can tweak it to whatever you prefer.

Comment (cah-nathanzender) [reply src/rules/nameCasing.test.js:4]: Added an editorconfig in #14 and will reformat all these new files i added.

**Comment by kristiandupont** [comment]
Great! I’m on vacation this week but I will merge this when I’m back by my
computer next week :-)

**Comment by zendern** [comment]
@kristiandupont sounds good.... Enjoy your vacation!!! 👋

**Pull request merged**